tools/astarte_export/lib/astarte/export.ex

```diff
@@ -318,12 +324,52 @@ defmodule Astarte.Export do
     generate_object_item_xml(fd, state, t)
   end
 
+  def process_attributes(device_data, fd, state) do
+    with attributes = device_data.attributes,
+         {:ok, state} <- XMLGenerate.xml_write_start_tag(fd, {"attributes", []}, state),
```

Comment:
nit: the first instruction of this with is a match, so it can be safely put above the with


```suggestion
    attributes = device_data.attributes,
    with {:ok, state} <- XMLGenerate.xml_write_start_tag(fd, {"attributes", []}, state),
```